fluentbit/fluentbit.conf

```diff
@@ -6,13 +7,36 @@
     Tag cloud-init-output
     Key message
 
+[FILTER]
+    Name modify
+    Match cloud-init-output
+    Add Source cloud-init-output
```

Comment:
I haven't added a `Source` for the application logs - not sure if we want one if it's understood that the 'default' `Source` of logs in ELK would be application logs?